cmd/debug.go

```diff
@@ -1,23 +1,48 @@
 package cmd
 
 import (
+	"encoding/json"
+	"errors"
 	"fmt"
 	"log"
 	"net/http"
 	"os"
 	"runtime"
+	"strings"
 	"time"
 
 	"github.com/codegangsta/cli"
 	"github.com/exercism/cli/config"
 )
 
+type release struct {
+	Location string `json:"html_url"`
+	TagName  string `json:"tag_name"`
+}
+
+func (r *release) Version() string {
+	return strings.TrimPrefix(r.TagName, "v")
+}
+
 // Debug provides information about the user's environment and configuration.
 func Debug(ctx *cli.Context) {
 	defer fmt.Printf("\nIf you are having trouble and need to file a GitHub issue (https://github.com/exercism/exercism.io/issues) please include this information (except your API key. Keep that private).\n")
 
+	client := http.Client{Timeout: 5 * time.Second}
+
 	fmt.Printf("\n**** Debug Information ****\n")
 	fmt.Printf("Exercism CLI Version: %s\n", ctx.App.Version)
+
+	rel, err := checkLatestRelease(client)
+	if err != nil {
+		log.Println(err)
+	} else {
+		if rel.Version() != ctx.App.Version {
+			defer fmt.Println("\nYour CLI is outdated. A new release can be found here:", rel.Location)
```

Comment:
Maybe _There is a more recent version of the CLI here:_ instead of saying that yours is outdated. Or maybe _A newer version of the CLI (vX.Y.Z) can be downloaded here:_